kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/signature/ContainerSignatureValidationService.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.container.signature;
+
+import java.util.Optional;
+import org.eclipse.kura.configuration.Password;
+import org.eclipse.kura.container.orchestration.ImageInstanceDescriptor;
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * Interface representing a service for validating the signature of a container image
+ *
+ * @since 2.3
```

Comment:
I'm not sure the version is correct. It should match the bundle version as far as I know

Reply:
Should be addressed in 43d61a230ecfa9871a9bc18ab9c88653fc9c7363